### PR TITLE
Improve descriptions of the query routing metrics

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1409.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1409.surge.sh/operations-manual/5/monitoring/metrics/reference)

File: modules/ROOT/pages/monitoring/metrics/reference.adoc [hunk 1]
```diff
@@ -274,17 +274,13 @@ By default, database metrics include:
...
-|The total number of queries executed locally. (counter)
+|<prefix>.dbms.routing.query.count.local|The total number of queries executed locally. (counter)
 
-|<prefix>.dbms.routing.query.count.remote_internal
-|The total number of queries executed remotely to a member of the same cluster. (counter)
+|<prefix>.dbms.routing.query.count.remote_internal|The total number of queries routed over to another member of the same cluster. (counter)
 
-|<prefix>.dbms.routing.query.count.remote_external
-|The total number of queries executed remotely to a member of a different cluster. (counter)
+|<prefix>.dbms.routing.query.count.remote_external|The total number of queries routed over to a server outside the cluster, i.e. DBMS. (counter)
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:283]: What do you mean by ` i.e. DBMS.`?

**Review by tobias-johansson — APPROVED** [review]

File: modules/ROOT/pages/monitoring/metrics/reference.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/monitoring/metrics/reference.adoc:283]: let's remove it. I don't think that this metric can be applied to a single server.

Comment (NataliaIvakina) [inline modules/ROOT/pages/monitoring/metrics/reference.adoc:283]:
```suggestion
|<prefix>.dbms.routing.query.count.remote_external|The total number of queries routed over to a server outside the cluster. (counter)
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1409.surge.sh

Updated pages:
- [Metrics reference](https://neo4j-docs-operations-1409.surge.sh/operations-manual/5/monitoring/metrics/reference)

**Pull request merged**